internal/outagelog/outagelog_test.go

```diff
@@ -0,0 +1,114 @@
+package outagelog_test
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"log/slog"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/EvergenEnergy/remote-standby/internal/config"
+	"github.com/EvergenEnergy/remote-standby/internal/mqtt"
+	"github.com/EvergenEnergy/remote-standby/internal/outagelog"
+	"github.com/EvergenEnergy/remote-standby/internal/publisher"
+	"github.com/EvergenEnergy/remote-standby/internal/standby"
+	"github.com/EvergenEnergy/remote-standby/internal/storage"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	testLogger     = slog.New(slog.NewTextHandler(os.Stdout, &slog.HandlerOptions{Level: slog.LevelDebug}))
+	defaultTimeout = 10 * time.Second
+)
+
+func getTestConfig() config.Config {
+	tmpLogPath := fmt.Sprintf("/tmp/outage-%d.log", time.Now().Nanosecond())
+	return config.Config{
+		SiteName:     "test",
+		SerialNumber: "device",
+		MQTT: config.MQTTConfig{
+			BrokerURL:         "tcp://localhost:1883",
+			ReadCommandTopic:  "cmd/site/handler/serial/cloud",
+			WriteCommandTopic: "cmd/site/handler/serial/standby",
+			StandbyTopic:      "cmd/site/standby/serial/plan",
+		},
+		Standby: config.StandbyConfig{
+			CheckInterval:   time.Duration(1 * time.Second),
+			OutageThreshold: time.Duration(2 * time.Second),
+			OutageLogFile:   tmpLogPath,
+		},
+	}
+}
+
+func TestWriteOutageLog(t *testing.T) {
```

Comment:
Please try to stick to `MethodName_StateUnderTest_ExpectedBehavior` convention, like we did in `config_test`. The latter two parts are missing here.

Reply:
I'm happy to use this convention when it adds value but I think when there's nothing special about the state under test and expected behaviour (like for a basic happy path), it adds a lot of words without much benefit. Plus it distracts/conflicts with the suffix we use to mark Integration tests.

Reply:
I'm afraid that I have to disagree :) By following the convention, you're making an explicit unambiguous statement that it's a basic happy path, and no contributor in the future will have to deduce that on their own. I can see that the test performs a call to `logHandler.Close()`. Is that the necessary condition? Or just cleanup? Or can it be removed at all?

This convention is a great helper tool for being systematic. It forces you to test one method at a time, under one condition at a time. When it becomes uncomfortable to follow this convention, it's very likely an indication that some design decisions should be re-evaluated.

Characters don't cost anything. I wouldn't bother ever so slightly about my test names becoming "long", unless it's a side effect of cramming inherently unrelated stuff into them, caused by awkward design decisions. 

I'm leaving the decision up to you anyways, as I don't want to block the development.

Reply:
Thanks for the explanation. I'll leave this as is but will keep that in mind for the future.